cypress/integration/data-import/match-on-pol-and-update-items.spec.js

```diff
@@ -81,23 +81,23 @@ describe('ui-data-import: Match on POL and update related Instance, Holdings, It
         update: true },
       actionProfile: { typeValue: NewActionProfile.folioRecordTypeValue.instance,
         name: actionProfileNameForInstance,
-        action: 'Update (all record types except Orders, Invoices, or MARC Holdings, Invoices, or MARC Holdings)' }
+        action: 'Update (all record types except Orders, Invoices, or MARC Holdings)' }
     },
     {
       mappingProfile: { typeValue: NewMappingProfile.folioRecordTypeValue.holdings,
         name: mappingProfileNameForHoldings,
         update: true },
       actionProfile: { typeValue: NewActionProfile.folioRecordTypeValue.holdings,
```

Comment:
Are you sure that the items you are searching in this test exist before test execution?

Reply:
yes, it is created by creating an order